src/components/PageSizeSelector.tsx

```diff
@@ -1,80 +0,0 @@
-import styled from "@emotion/styled";
-import { Select, Option } from "@leafygreen-ui/select";
-import { Select as AntdSelect } from "antd";
-import { useHistory, useLocation } from "react-router-dom";
-import { PAGE_SIZES, RECENT_PAGE_SIZE_KEY } from "constants/index";
-import { queryString } from "utils";
-
-const { stringifyQuery, parseQueryString } = queryString;
-
-const { Option: AntdOption } = AntdSelect;
-
-interface Props {
-  value: number;
-  "data-cy"?: string;
-  onClick?: (i: number) => void;
-  sendAnalyticsEvent?: () => void;
-  useLeafygreen?: boolean;
-}
-
-export const PageSizeSelector: React.VFC<Props> = ({
-  value,
-  "data-cy": dataCy,
-  onClick,
-  sendAnalyticsEvent = () => undefined,
-  useLeafygreen = false,
-}) => {
-  const { replace } = useHistory();
-  const { search, pathname } = useLocation();
-
-  const handleChange =
-    onClick ||
-    ((pageSize: number) => {
```

Comment:
I kind of liked this pattern, and was wondering if there's a reason we went away from it

I thought it made sense that `PageSizeSelector` could call an optional `onChange` function if it’s defined, otherwise it can just always default to using `setPageSize`. Similarly the analytics event could be another optional prop function. 

```
const handlePageSize = (pageSize: number) => {
	if (onChange) {
	    onChange(pageSize);
	} else {
	    setPageSize(pageSize);
        }
	sendEvent?.({name: "Change Page Size"});
}
```

That way there'd be no reason to import `usePageSizeSelector` across multiple places in the codebase, it could just be imported in `PageSizeSelector`